verified-inference/host/traffic_forwarder.py

```diff
@@ -0,0 +1,93 @@
+import socket
+import struct
+import threading
+import logging
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(message)s")
+logger = logging.getLogger()
+
+
+class HostProxy:
+    BUFFER_SIZE = 1024
+    CID = 3  # The CID of the enclave
```

Comment:
How could the enclave CID be a constant? There should also be multiple CIDs for multiple enclaves, isn't it?

Reply:
yep, but according to the usage, the `vsock_server.bind((self.CID, self.PORT))` should be connecting to the enclave IIUC. so here shouldn't we provide the enclave CID?